db/migrate/20200418113601_load_schema.rb

```diff
@@ -0,0 +1,404 @@
+class LoadSchema < ActiveRecord::Migration
+  def up
+    # These are extensions that must be enabled in order to support this database
+    enable_extension "plpgsql"
+
+    create_table "contest_relations", force: :cascade do |t|
+      t.integer  "user_id"
```

Comment:
Nit: It would be nice if these lines were naturally indented the same as in schema.rb (so they can be copy pasted directly without re-indenting). How about this hack:

```ruby
class LoadSchema < ActiveRecord::Migration
  def up
    define_schema
    
    execute "CREATE UNIQUE INDEX index_users_on_username ON users (lower(username))"
  end
  
  def down
    ...
  end
end

# We define this instance method outside of the class, so the body will be
# indented the same as the code in db/seed.rb.
LoadSchema.send(:define_method, :define_schema) do
    create_table "contest_relations", force: :cascade do |t|
      ...
    end
end
```